ui/warrior/sim.ts

```diff
@@ -0,0 +1,187 @@
+		super(parentElem, player, {
+			cssClass: 'warrior-sim-ui',
+			// List any known bugs / issues here and they'll be shown on the site.
+			knownIssues: [
+			],
+
+			// All stats for which EP should be calculated.
+			epStats: [
+				Stat.StatStrength,
+				Stat.StatAgility,
+				Stat.StatAttackPower,
+				Stat.StatExpertise,
+				Stat.StatMeleeHit,
+				Stat.StatMeleeCrit,
+				Stat.StatMeleeHaste,
+				Stat.StatArmorPenetration,
+			],
+			// Reference stat against which to calculate EP. I think all classes use either spell power or attack power.
+			epReferenceStat: Stat.StatAttackPower,
+			// Which stats to display in the Character Stats section, at the bottom of the left-hand sidebar.
+			displayStats: [
+				Stat.StatStamina,
+				Stat.StatStrength,
+				Stat.StatAgility,
+				Stat.StatAttackPower,
+				Stat.StatExpertise,
+				Stat.StatMeleeHit,
+				Stat.StatMeleeCrit,
+				Stat.StatMeleeHaste,
+				Stat.StatArmorPenetration,
+			],
+
+			defaults: {
+				// Default equipped gear.
+				gear: Presets.P1_FURY_PRESET.gear,
+				// Default EP weights for sorting gear in the gear picker.
+				epWeights: Stats.fromMap({
+					[Stat.StatStrength]: 2.5,
+					[Stat.StatAgility]: 1.75,
+					[Stat.StatAttackPower]: 1,
+					[Stat.StatExpertise]: 3.75,
+					[Stat.StatMeleeHit]: 1.5,
+					[Stat.StatMeleeCrit]: 2.5,
+					[Stat.StatMeleeHaste]: 3,
+					[Stat.StatArmorPenetration]: 0.5,
+				}),
+				// Default consumes settings.
+				consumes: Presets.DefaultFuryConsumes,
+				// Default rotation settings.
+				rotation: Presets.DefaultFuryRotation,
+				// Default talents.
+				talents: Presets.FuryTalents.data,
+				// Default spec-specific settings.
+				specOptions: Presets.DefaultFuryOptions,
+				// Default raid/party buffs settings.
+				raidBuffs: RaidBuffs.create({
+					giftOfTheWild: TristateEffect.TristateEffectImproved,
+				}),
+				partyBuffs: PartyBuffs.create({
+					drums: Drums.DrumsOfBattle,
+					bloodlust: 1,
+					strengthOfEarthTotem: StrengthOfEarthType.EnhancingAndCyclone,
+					windfuryTotemRank: 5,
+				}),
+				individualBuffs: IndividualBuffs.create({
+					blessingOfKings: true,
+					blessingOfMight: TristateEffect.TristateEffectImproved,
+					innervates: 1,
+				}),
+				debuffs: Debuffs.create({
+					bloodFrenzy: true,
+					curseOfRecklessness: true,
+					exposeArmor: TristateEffect.TristateEffectRegular
+				}),
+			},
+
+			// IconInputs to include in the 'Self Buffs' section on the settings tab.
+			selfBuffInputs: [
+				// TODO: Move reck to cooldown tabs
+				WarriorInputs.Recklessness ,
+				IconInputs.DrumsOfBattleConsume,
+			],
+			// IconInputs to include in the 'Other Buffs' section on the settings tab.
+			raidBuffInputs: [
+				IconInputs.GiftOfTheWild,
+			],
+			partyBuffInputs: [
+				IconInputs.LeaderOfThePack,
+				IconInputs.DrumsOfBattleBuff,
+				IconInputs.Bloodlust,
+				// IconInputs.WindfuryTotem,
+				// IconInputs.StrengthOfEarthTotem,
+				// IconInputs.UnleashedRage,
```

Comment:
I'll try to get these (and the consume ones below) added sometime this week!

Reply:
It'd be rad if you could also include the missing gems such as Smooth Dawnstone and Jagged Talasite.